src/main/resources/application.properties

```diff
@@ -137,7 +137,7 @@ ccd.draft.encryptionKey=${CCD_DRAFT_ENCRYPTION_KEY:xxxxxxxxxxxxxxxx}
 ccd.draft.maxTTLDays=${CCD_DRAFT_TTL_DAYS:180}
 
 # Search
-search.elastic.enabled=${ELASTIC_SEARCH_ENABLED:false}
+search.elastic.enabled=${ELASTIC_SEARCH_ENABLED:true}
```

Comment:
why change this?

Reply:
Reverted the changes. It was by mistake.